archivist/notebooks/Find Asset and Create Attachment.ipynb

```diff
@@ -0,0 +1,331 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a93631ae",
+   "metadata": {},
+   "source": [
+    "## Find Asset and Create Attachment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "0b61044f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Find existing Asset and create an attachment\n",
+    "#\n",
+    "# Main function, establishes a connection to RKVST using an App Registration then uses that\n",
+    "# to find existing Asset and create an attachment.\n",
+    "#\n",
+    "# The attachment is an image located in a folder named \"test_files\"\n",
+    "#\n",
+    "# Note: The purpose of RKVST Jupyter Notebooks is to provide simplified examples that one can easily execute and digest.\n",
+    "# The RKVST Python SDK is authored to work cleanly with more advanced coding techniques.\n",
+    "#\n",
+    "# RKVST Python SDK: https://github.com/rkvst/rkvst-python\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "f9e2c868",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from json import dumps as json_dumps\n",
+    "from os import getenv\n",
+    "from warnings import filterwarnings\n",
+    "\n",
+    "from archivist.archivist import Archivist\n",
+    "from archivist.proof_mechanism import ProofMechanism\n",
+    "from archivist.logger import set_logger\n",
+    "\n",
+    "filterwarnings(\"ignore\", message=\"Unverified HTTPS request\")\n",
```

Comment:
Do we need this? Obviously on RKVST scratch installations it will trigger but all official SaaS should have good TLS.

Reply:
Tested a notebook and removed filterwarnings statement and it worked w/out the extra warning verbiage.  Removed from notebooks